pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -199,12 +199,16 @@ KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/k
 .PHONY: kustomize
 kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
 $(KUSTOMIZE): $(LOCALBIN)
+	@if test -x $(LOCALBIN)/kustomize && ! $(LOCALBIN)/kustomize version | grep -q $(KUSTOMIZE_VERSION); then
```

Comment:
Could we update the descriptions to make clear that if is not the version then we remove and re-download them

Reply:
done